tests/test_releases.py

```
@@ -37,6 +38,66 @@ def test_apply_update_replaces_files_and_leaves_legacy_backups_untouched(tmp_pat
+
+    releases.ReleasesRoutes._apply_update(archive)
+
+    assert (root / "main.py").read_text(encoding="utf-8") == "new application"
+    assert (root / "data" / "config" / "settings.json").read_text(encoding="utf-8") == "default user settings"
+    assert (runtime_data / "config" / "settings.json").read_text(encoding="utf-8") == "user settings"
+
+
+def test_repository_does_not_track_runtime_data() -> None:
+    project_root = Path(__file__).resolve().parents[1]
+    tracked_paths = subprocess.run(
+        ["git", "ls-files", "--", "data"],
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)